builtin/providers/aws/resource_aws_lambda_function.go

```diff
@@ -270,6 +284,45 @@ func resourceAwsLambdaFunctionRead(d *schema.ResourceData, meta interface{}) err
 	}
 	d.Set("source_code_hash", function.CodeSha256)
 
+	// List is sorted from oldest to latest
+	// so this may get costly over time :'(
+	var lastVersion, lastQualifiedArn string
```

Comment:
:( My eyes are bleeding to know that you had to do that and also go to the last page or results LOL